app/assets/stylesheets/stores.css.scss

```diff
@@ -0,0 +1,21 @@
+// Place all the styles related to the stores controller here.
+// They will automatically be included in application.css.
+// You can use Sass (SCSS) here: http://sass-lang.com/
+// .store{
+//   border: 5px;
+// }
+
+// .store{
+//   th{
+//     border: 5px;
+//   }
+//
+//   td{
+//     border: 5px;
+//   }
+// }
+
+table, td, th {
+    border: 1px solid #000;
```

Comment:
Color literals like `#000` should only be used in variable declarations; they should be referred to via variable everywhere else.<br>Line should be indented 2 spaces, but was indented 4 spaces